test/e2e/performanceprofile/functests/0_config/config.go

```diff
@@ -3,6 +3,7 @@ package __performance_config
 import (
 	"context"
 	"fmt"
+	"k8s.io/utils/cpuset"
```

Comment:
minor: wrong grouping of imports. This group should only contain stdlib imports

Reply:
I keep missing that, will fix it thank you for the note.